### PR TITLE
Update project

### PR DESCRIPTION
**Pull request by friederbluemle**
A bunch of whitespace cleanup, and minor wording/formatting updates (separate commits).

<sub><sup>Please **NO GitHub rebase/squash merges**. The changes in this pull request are broken up into meaningful, [atomic commits](https://www.freshconsulting.com/atomic-commits/), and my branch should already be up-to-date with the latest target branch. If it isn't, or if you'd like me to combine some or all of the commits, please let me know, and I will make the necessary updates as soon as possible. Thank you!</sup></sub>

### CODE REVIEW

**Comment by friederbluemle** [comment]
@x2on Could you please have a look at this PR when you get a chance? Thanks

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1 @@
 language: groovy
-
```
Comment (x2on) [inline .travis.yml]: Why is this removed?

Comment (friederbluemle) [reply .travis.yml]: Good question, and sorry for not explaining it in the PR description.

    sudo: false

**Reason for removal**: `sudo: false` has been the *default* on Travis CI since they switched over to container-based infrastructure a few years ago.

    install: ./gradlew clean test assemble

**Reason for removal**: Travis CI automatically executes `assemble` to build, and `check` to test a Gradle-based project. `check` also runs `test`, there is no need to `clean` - it's always a fresh build on Travis anyway.

    branches:
      only:
        - master

**Reason for removal**: You *could* keep this, but what's the benefit of *not* having automatic (and free) CI feedback from Travis CI?

    env:
       - TERM=dumb

**Reason for removal**: It is possible that this was needed in the past, but apparently it's no longer necessary. You can see the correctly formatted log output when you check the successful build of this PR here:
https://travis-ci.org/friederbluemle/gradle-hockeyapp-plugin

Also see:
https://docs.travis-ci.com/user/languages/groovy/#Overview
> [...] reasonable defaults, so quite often you won’t have to configure anything beyond: `language: groovy`

**Comment by x2on** [comment]
Thanks for your PR and sorry for the delay
